platform/ios/src/MGLCalloutView.h

```diff
@@ -36,6 +36,18 @@ NS_ASSUME_NONNULL_BEGIN
 @property (nonatomic, weak) id<MGLCalloutViewDelegate> delegate;
 
 /**
+ A Boolean value indicating whether the callout view should be positioned automatically.
+ The exact position can be adjusted by overriding -[UIView setCenter:]
+ */
+@property (nonatomic, assign) BOOL automaticPositioning;
+
+/**
+ A Boolean value indicating whether the callout view should be dismissed or not
+ when the map view changes viewport.
+ */
+@property (nonatomic, assign) BOOL staysOpen;
```

Comment:
Can we reverse the logic and call it `dismissesAutomatically`? I think that would make the behavior clearer, as evidenced by the description you wrote for this property.